src/Uno.Toolkit.RuntimeTests/Tests/AutoLayoutTest.cs

```diff
@@ -158,9 +156,13 @@ public async Task When_AbsolutePosition_WithPadding(bool isStretch, Orientation
 		var border2Transform = (MatrixTransform)border2.TransformToVisual(SUT);
 		var border3Transform = (MatrixTransform)border3.TransformToVisual(SUT);
 
+		//Element with border in wasm don't give the same result for TransformToVisual
+		//https://github.com/unoplatform/uno/issues/11410
+		//var offset = RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER")) ? 2 : 0;
+
 		if (orientation is Orientation.Vertical)
 		{
-			Assert.AreEqual(border1Transform!.Matrix.OffsetY!, rec1expected);
+			Assert.AreEqual(border1Transform!.Matrix.OffsetY!, rec1expected); // + offset);
```

Comment:
if you fixed the root cause, you can remove these comments/unused code

Reply:
thanks